tests/gene-comparison-tool.spec.ts

```diff
@@ -27,4 +27,46 @@ test.describe('specific viewport block', () => {
     const dropdown = page.locator('#subCategory');
     await expect(dropdown).toHaveText('Targeted Selected Reaction Monitoring (SRM)');
   });
+
+  test('switching from RNA to Protein with RNA-specific column ordering reverts back to Risk Score descending', async ({ page }) => {
+    // set category for Protein - Differential Expression
+    await page.goto('/genes/comparison?sortField=FP&sortOrder=1');
```

Comment:
Could use a newly added constant here -- 

```suggestion
    // set category for Protein - Differential Expression
    await page.goto(`${URL_GCT}?sortField=FP&sortOrder=1`);
```

Reply:
good idea